src/sql/base/browser/ui/panel/panel.ts

```diff
@@ -182,6 +184,12 @@ export class TabbedPanel extends Disposable {
 		this._shownTabId = id;
 		this.tabHistory.push(id);
 		const tab = this._tabMap.get(this._shownTabId)!; // @anthonydresser we know this can't be undefined since we check further up if the map contains the id
+
+		if (tab.containsWebView && tab.body) {
```

Comment:
this iteration of the change rebuilds the webview when needed.  I'm going to add the retainWhenHidden functionality separately to get these fixes out sooner.